src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -206,15 +206,23 @@ public SscsCaseData getOldSscsCaseData() {
             || DIRECTION_ISSUED_WELSH.equals(getNotificationType())
             || DWP_UPLOAD_RESPONSE_NOTIFICATION.equals(getNotificationType())
             || EVIDENCE_REMINDER_NOTIFICATION.equals(getNotificationType())
-            || REQUEST_INFO_INCOMPLETE.equals(getNotificationType())
+            || (REQUEST_INFO_INCOMPLETE.equals(getNotificationType()) && responseWrapper.getNewSscsCaseData().getInformationFromPartySelected() != null && PartyItemList.JOINT_PARTY.getCode().equals(responseWrapper.getNewSscsCaseData().getInformationFromPartySelected().getValue().getCode()))
             || JOINT_PARTY_ADDED.equals(getNotificationType())
+            || isValidRequestInfoIncompleteEventForParty(PartyItemList.JOINT_PARTY)
```

Comment:
Isn't this the same as line 209?

Reply:
Yeh line 209 should have been removed as it does the same as this line. Updated.